src/main/java/frc/robot/subsystems/climber/C2025ClimberIO.java

```diff
@@ -98,4 +98,8 @@ public void setTargetVoltage(double voltage) {
         motor.setControl(voltageRequest.withOutput(voltage));
     }
 
+    @Override
+    public Climber.Constants getConstants() {
+        return new Climber.Constants(1.0);
```

Comment:
The climber should be a `private static final` constant field in the `C2025ClimberIO`. Similar to how it is in the elevator or arm.